src/dashboard/src/components/administration/forms.py

```diff
@@ -64,6 +64,16 @@ class StorageSettingsForm(SettingsForm):
         label="Full URL of the storage service")
 
 
+class ChecksumSettingsForm(SettingsForm):
+    CHOICES = (
+        ('md5', 'MD5'),
```

Comment:
Was MD5 specifically requested? Both it and SHA1 feel a bit insecure at this point.